web/src/app/administration/components/access-items-management/access-items-management.component.ts

```diff
@@ -163,18 +161,15 @@ export class AccessItemsManagementComponent implements OnInit, OnDestroy {
         // new Key: ALERT_TYPES.SUCCESS_ALERT
         response => {
           this.requestInProgressService.setRequestInProgress(false);
-          this.alertService.triggerAlert(
-            new AlertModel(
-              AlertType.SUCCESS,
-              `${this.accessIdSelected
-              } was removed successfully`
-            )
+          this.notificationsService.triggerAlert(
+            NOTIFICATION_TYPES.SUCCESS_ALERT, new Map<string, string>([['this.accessIdSelected', this.accessIdSelected]])
```

Comment:
can we replace the `this.` from the replacement?

Reply:
done